tests/core/arch-arm/stalker-arm.c

```diff
@@ -1767,6 +1768,27 @@ TESTCASE (thumb_it_eq)
   g_assert_cmpuint (fixture->sink->events->len, ==, 0);
 }
 
+TESTCODE (thumb_it_al,
+  0x00, 0xb5, /* push {lr}       */
+  0x00, 0x1a, /* subs r0, r0, r0 */
+  0xe8, 0xbf, /* it al           */
+  0x01, 0x30, /* adds r0, #1     */
+
+  /* part_two:                   */
+  0xe8, 0xbf, /* it al           */
+  0x02, 0x30, /* adds r0, #2     */
+  0x04, 0x30, /* adds r0, #4     */
+  0x00, 0xbd, /* pop {pc}        */
+);
+
+TESTCASE (thumb_it_al)
+{
+  INVOKE_THUMB_EXPECTING (GUM_NOTHING, thumb_it_al, 7);
+
+  g_assert_cmpuint (fixture->sink->events->len, ==, 0);
+}
+
+
```

Comment:
Left-over blank line. (To match the existing style in frida-gum, blank lines should never follow each other.)